config/config-api/src/com/thoughtworks/go/config/PipelineConfig.java

```diff
@@ -164,7 +168,11 @@ private void validateStageNameUniqueness() {
 
     private void validateLabelTemplate() {
         if (XmlUtils.doesNotMatchUsingXsdRegex(LABEL_TEMPLATE_FORMAT_REGEX, labelTemplate)) {
-            addError("labelTemplate", "Invalid label. Label should be composed of alphanumeric text, it should contain material revision or ${COUNT}, or use params as #{<param-name>}.");
+            addError("labelTemplate",
```

Comment:
could be help text [here](https://github.com/gocd/gocd/blob/master/server/webapp/WEB-INF/rails.new/app/views/admin/pipelines/general.html.erb#L16)?